docs/charra-concept-on-pc.md

```diff
@@ -0,0 +1,28 @@
+# How to run CHARRA concept on PC
+
+The detailed steps of how to build and run CHARRA applications (Attester and Verifier) are
+discribed on corresponding GitHub repository:
+ - [Build and run CHARRA on PC](https://github.com/Fraunhofer-SIT/charra#build-and-run)
```

Comment:
I think you should briefly describe the exact steps you used to build Charra container.
Theirs documentation isn't 100% clear, e.g. documentation says to tag the TPM emulator container `tpm2software/tpm2-tss:ubuntu-20.04` but I had to tag it `ghcr.io/tpm2-software/ubuntu-20.04:latest` to build Charra container.

Reply:
Hm... just run `./docker/build.sh` and necessary packages were installed in container